packages/grpc-js-xds/interop/Dockerfile

```
@@ -0,0 +1,30 @@
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Dockerfile for building the xDS interop client. To build the image, run the
+# following command from grpc-node directory:
+# docker build -t <TAG> -f packages/grpc-js-xds/interop/Dockerfile .
+
+FROM node:16-alpine
+
+# Make a grpc-node directory and copy the repo into it.
+WORKDIR /node/src/grpc-node
+COPY . .
+
+WORKDIR /node/src/grpc-node/packages/grpc-js
+RUN npm install
+WORKDIR /node/src/grpc-node/packages/grpc-js-xds
+RUN npm install
+
+ENTRYPOINT [ "node", "/node/src/grpc-node/packages/grpc-js-xds/build/interop/xds-interop-client" ]
```

I think the issue was with using `WORKDIR` multiple times. WORKDIR sets the path relative to which the rest of commands are executed. Consider this:

```suggestion
RUN cd packages/grpc-js && npm install
RUN cd packages/grpc-js-xds && npm install

ENTRYPOINT [ "node", "packages/grpc-js-xds/build/interop/xds-interop-client" ]
```

---

I am pretty sure that this doesn't matter now that my entrypoint is an absolute path.